pygmt/tests/test_plot3d.py

```diff
@@ -329,6 +318,34 @@ def test_plot3d_colors_sizes_proj(data, region):
     return fig_ref, fig_test
 
 
+@pytest.mark.mpl_image_compare
+def test_plot3d_varying_intensity():
+    """
+    Plot the data with array-like intensity.
+    """
+
+    x = np.arange(-1, 1.1, 0.1)
+    y = np.zeros(x.size)
+    z = y
+    intensity = x
+
+    fig = Figure()
+    fig.plot3d(
+        x=x,
+        y=y,
+        z=z,
+        region=[-1.1, 1.1, -0.5, 0.5, -0.5, 0.5],
+        projection="x15c",
```

Comment:
Any particular reason that you want to use `x15c` (not `X15c`) here?
```suggestion
        projection="x15c",
```

Reply:
Revised in 3c12707.